packages/bpk-react-utils/src/TransitionInitialMount.js

```diff
@@ -33,19 +34,21 @@ const FirstChild = (props) => {
 const TransitionInitialMount = ({
   appearClassName, appearActiveClassName, transitionTimeout, children,
 }) => (
-  <CSSTransitionGroup
+  <TransitionGroup
     component={FirstChild}
-    transitionName={{
-      appear: appearClassName,
-      appearActive: appearActiveClassName,
-    }}
-    transitionAppear
-    transitionAppearTimeout={transitionTimeout}
-    transitionEnterTimeout={0}
-    transitionLeaveTimeout={0}
   >
-    {children}
-  </CSSTransitionGroup>
+    <CSSTransition
```

Comment:
@sogame I've read up on the v2 api - couldn't `CSSTransition` be used solely in this example too (like you have done with `BpkImage`)?

Reply:
I added it to preserve the prop `component={FirstChild}`, not sure if this can be killed...

Reply:
The `component={FirstChild}` code is a workaround ;) it seems like `CSSTransition ` is a much better fit now.

Reply:
Cool! I'll kill it then :-)

Reply:
Killed with fire! :-D

I've just pushed the changes :-)